### DISCUSSION TITLE
fabric 官方qq群支持

### DISCUSSION THREAD

**Discussion by baihuamen** [FabricMC/fabric-api#4282] [FAQ]
目前中国大陆已经无法访问discord，改hosts也没用，建议fabric官方建一个qq群来方便中国大陆开发者交流

**Comment by Earthcomputer** [comment]
Discussions in English only, please. You can access Discord from within China by using a VPN.

**Reply by baihuamen** [reply]
I know,but I hope that some chinese fabric contributor can set up a qq group as a official discussion group in China .(it's unbelievable that there hasn't Chinese contributor)

**Reply by Cai-kunkun** [reply]
But using VPN is illegal in mainland China. I think we can create a KOOK channel,its ui likes discord. And non china devs can access this channel to develop mods together.

**Reply by baihuamen** [reply]
> But using VPN is illegal in mainland China. I think we can create a KOOK channel,its ui likes discord. And non china devs can access this channel to develop mods together.
But the most difficult is there hasn't any member or contributor to create it.

**Reply by Cai-kunkun** [reply]
Yes , my grammer has some wrong. I am sorry about that.

**Reply by Earthcomputer** [reply]
Nobody is stopping Chinese developers from making their own unofficial mod dev channels

**Reply by Cai-kunkun** [reply]
fine :(

**Reply by baihuamen** [reply]
Yes,but unofficial make those channels dispersion.

**Reply by Cai-kunkun** [reply]
Yes thats the biggest issue=w=

**Reply by baihuamen** [reply]
> Yes thats the biggest issue=w=

I found a win-win solution ,telegram  and whatsapp is available in china .(telegram  seem more stable than whatsapp)I think it's a good idea to start a new official channel  by them.And it won't  make a legal problem  if we use them comform to China's laws .
